terminal_ui.py

```diff
@@ -0,0 +1,46 @@
+from game import Game
+
+
+class TerminalUI:
+    def __init__(self):
+        self.play()
+
+    def play(self):
+        game.set_up_game()
+        self.print_board()
+        while not game.winner:
+            self.make_move()
+        print('You won!')
+
+    def make_move(self):
+        self.print_score()
+        row, column = self.ask_for_move()
+        game.play_game(row, column)
+        self.print_board()
+        if game.reset_board:
+            game.set_up_board()
+            self.print_board('Sorry, no more move! \nThis is your new board: ')
+
+    @staticmethod
+    def print_board(text='This is the current board: '):
+        print(text)
+        for row in game.board:
+            print(row)
+
+    @staticmethod
+    def print_score():
+        print('Your target is {} points. You currently earned {} points.'.format(game.target, game.score))
```

Comment:
we can use f-strings for better readability.
`print(f'Your target is {game.target} points. You currently earned {game.score} points.')`

Reply:
Ah, I didn't know I could do it like that 😄 , changed